tools/eval/README.md

```diff
@@ -190,4 +191,18 @@ Run evaluation with the following command:
 
 ```shell
 python eval.py -m crnn -d iiit5k -dr /path/to/iiit5k
+```
+
+
+## PPHumanSeg
```

Comment:
I wonder if we can use the `mini_supervisely` for the name of dataset so as to avoid confusion with the model itself.